lib/UUID4/Tiny.pm

```diff
@@ -1,9 +1,11 @@
-package UUID4::Tiny 0.001;
+package UUID4::Tiny;
 # ABSTRACT: Cryptographically secure v4 UUIDs for Linux x64
 
 use strict;
 use warnings;
 
+our $VERSION = '0.001';
```

Comment:
No changes needed, but there's also [OurPkgVersion](https://metacpan.org/pod/Dist::Zilla::Plugin::OurPkgVersion), which is very useful when the distribution has more than one file.

Reply:
Yes. Sadly managed_versions appears to be very strict about how you define the version so I had to move it out of the package declaration.